os_migrate/plugins/modules/export_security_group_rules.py

```diff
@@ -88,9 +88,9 @@ def run_module():
         sec_refs = network.security_group_rule_refs_from_sdk(conn, sec_rule_obj)
         ser_sec = network.serialize_security_group_rule(sec_rule_obj, sec_refs)
 
-    result['changed'] = filesystem.write_or_replace_resource(
-        module.params['path'], ser_sec)
+        filesystem.write_or_replace_resource(module.params['path'], ser_sec)
 
+    result['changed'] = True
```

Comment:
`result['changed']` should be True only if at least one of the write_or_replace_resource calls returned true